contrib/src/test/java/com/datatorrent/contrib/avro/SimpleOrder.java

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
It is possible to move it as a static class inside the test file, than creating a separate file?